### DISCUSSION TITLE
Typescript support

### DISCUSSION THREAD

**Discussion by StelyanBaltov** [jnsmalm/pixi3d#60] [General]
Hello, is there support for Typescript as well?

**Comment by jnsmalm** [comment]
Hey! Yes, there is support for TypeScript, the project itself is written in TypeScript.

**Comment by StelyanBaltov** [comment]
Thank you for the fast answer! Can you please make an example how to implement 3d models in a proper typescript project. Cheers!

**Reply by jnsmalm** [reply]
You can find example at https://codesandbox.io/s/github/jnsmalm/pixi3d-sandbox/tree/master/standard-material

Should be trivial to convert to TypeScript.
